### PR TITLE
chore: update the 'metamask-sdk' package to version '0.26.1'

### PR DESCRIPTION
**Pull request by omridan159**
### Description

Update the metamask-sdk package to version '0.26.1'

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

### CODE REVIEW

File: packages/metamask/package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@metamask/sdk": "^0.20.5",
+    "@metamask/sdk": "^0.26.0",
```
Comment (Adamj1232) [inline packages/metamask/package.json:63]: The package version on line 3 needs an alpha version bump and also needs to be reflected and tested in `packages/demo/package.json`

Comment (omridan159) [reply packages/metamask/package.json:63]: Screenshot 2024-06-21 at 10 25 25

Comment (omridan159) [reply packages/metamask/package.json:63]: After updating the MetaMask SDK package to version "2.1.3-alpha.1", I encounter an issue when I go to the demo test dapp and try to run yarn after also updating MetaMask there to "@web3-onboard/metamask": "^2.1.3-alpha.1". 

What is the problem?

File: packages/demo/package.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "@web3-onboard/keystone": "^2.3.8",
     "@web3-onboard/ledger": "^2.7.1",
     "@web3-onboard/magic": "^2.2.1",
-    "@web3-onboard/metamask": "^2.1.2-alpha.1",
+    "@web3-onboard/metamask": "^2.1.3-alpha.1",
```
Comment (Adamj1232) [inline packages/demo/package.json:51]:
```suggestion
    "@web3-onboard/metamask": "^2.1.2-alpha.2",
```

**Review by Adamj1232 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**